.Rbuildignore

```diff
@@ -10,3 +10,6 @@ test/
 ^\.github$
 ^LICENSE\.md$
 ignore/
+^doc$
+^Meta$
+ignore/
```

Comment:
Missing new line

Reply:
Arghhh, one day I will get this right!

Reply:
I've realised what I've done wrong in the past. I normally just hit 'Enter' and save the file but in my recent commit ba0faba8 I've hit enter twice and save to force a new line! A late lesson learned!